compiler/aarch64/codegen/OpBinary.cpp

```diff
@@ -389,6 +389,8 @@ const OMR::ARM64::InstOpCode::OpCodeBinaryEntry OMR::ARM64::InstOpCode::binaryEn
 		0x9BA08000,	/* UMSUBL    	umsubl	 */
 		0x9B400000,	/* SMULH     	smulh	 */
 		0x9BC00000,	/* UMULH     	umulh	 */
+		0X1F000000,	/* FMADD        fmaddd   */
+		0X1F400000,	/* FMADD        fmadds   */
```

Comment:
I think you have the double and single precision encodings reversed.  See:
<img width="510" alt="ss" src="https://user-images.githubusercontent.com/17713611/79596964-6deb7100-80af-11ea-9e2e-5e85cc2d01dd.png">

Reply:
Sorry, that is a mistake. I have fixed that in 945cc7a